src/coreclr/vm/ceeload.cpp

```diff
@@ -932,6 +932,35 @@ TADDR Module::GetDynamicIL(mdToken token)
     return entry.m_il;
 }
 
+#ifndef DACCESS_COMPILE
+void Module::SetDynamicRvaField(mdToken token, TADDR blobAddress)
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_NOTRIGGER;
+        MODE_COOPERATIVE;
+    }
+    CONTRACTL_END;
+
+    // Reuse existing dynamic IL mechanism to store/map the data.
+    SetDynamicIL(token, blobAddress);
```

Comment:
Might want to add comment on SetDynamicIL that mentions the token/data can now also be field rva.